.github/CODEOWNERS

```diff
@@ -15,6 +15,7 @@
 /filter-hitl-language @holtskinner
 /fraud-detection-python @holtskinner
 /tax-processing-pipeline-python @holtskinner
+/bq-connector @mservidio
```

Comment:
@holtskinner Should I remove this? I assume I would need access to the repo itself to be a code owner?

Reply:
You can keep it, just request to join https://github.com/orgs/GoogleCloudPlatform/teams/document-ai-samples-contributors/members